processing/src/main/java/io/druid/segment/IndexMerger.java

```diff
@@ -58,6 +60,10 @@
 import java.util.Set;
 import java.util.stream.Collectors;
 
+@JsonTypeInfo(use = JsonTypeInfo.Id.NAME, property = "type", defaultImpl = IndexMergerV9.class)
+@JsonSubTypes(value = {
+    @JsonSubTypes.Type(name = "v9", value = IndexMergerV9.class)
+})
 @ImplementedBy(IndexMergerV9.class)
```

Comment:
`@ImplementedBy` is guice thing, I'm not sure it can work properly after removing it if it is not a json case